src/components/reusable/textArea/textArea.scss

```diff
@@ -24,11 +24,14 @@ textarea {
   right: 16px;
 }
 
-.count {
-  @include typography.type-ui-03;
-  position: absolute;
-  right: 22px;
-  bottom: 6px;
-  padding: 2px;
-  background: var(--kd-color-background-inverse);
+.caption-error-count {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  .count {
+    @include typography.type-ui-03;
+    margin-top: 13px;
+    margin-left: auto;
+    background: var(--kd-color-background-inverse);
```

Comment:
Seems to be no need for these properties any longer.

Reply:
Removed  margin-left and background . 
Retained margin-top to prevent the shifting of count div in absence of caption/invalidText.